examples/gno.land/p/gov/proposal/types.gno

```diff
@@ -0,0 +1,14 @@
+// Package proposal defines types for proposal execution.
+package proposal
+
+// Executor represents a minimal closure-oriented proposal design.
+// It is intended to be used by a govdao governance proposal (v1, v2, etc).
+type Executor interface {
```

Comment:
Hum, if this is about performance, I guess that on Gno it's okay since we're mostly loading everything lazily.

Moving to another package could work, but I dislike packages named "types" because you need to name them explicitly in the import line from the caller contract, and also resulting in two imports instead of one.